cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1127,6 +1151,13 @@
           ]
         }
       ]
+    },
+    "UseHyperthreading": {
+      "Fn::Equals": [
+        -1, {
```

Comment:
Put `-1` second in the list, also run `tox -e cfn-format` on this file

Reply:
Changed